### DISCUSSION TITLE
How to adjust transmission power settings in Sionna?

### DISCUSSION THREAD

**Discussion by communicationDL1** [NVlabs/sionna#302] [General]
Hello,I want to investigate the impact of transmission power on communication systems, but I couldn't find the corresponding power settings in the relevant tutorials. How can I achieve this in Sionna?

**Comment by jhoydis** [comment]
Can you provide a bit of context to your question? Does it related to Sionna RT or the stochastic channel models?

In general, channel impulse responses as well as coverage maps are independent of the transmit power. 
For link-level simulations, we typically configure the effective signal-to-noise-ratio that accounts for bandwidth, noise and transmit power.

**Reply by communicationDL1** [reply]
Thank you for your reply! I want to observe the impact of transmission power in this example provided by Sionna :https://nvlabs.github.io/sionna/examples/MIMO_OFDM_Transmissions_over_CDL.html. Could you help me adjust the transimission power in this example?

**Reply by jhoydis** [reply]
The transmit power in this example is controlled indirectly by changing the noise power. 
So you can either scale the transmit signal ``x`` and keep the noise power fixed, or scale noise power ``no`` and keep the transmit power fixed. The only quantity that matters is the signal-to-noise ratio (SNR).

The BER-curves show the performance as a function of SNR.
Please see the following line in this [cell](https://nvlabs.github.io/sionna/examples/Realistic_Multiuser_MIMO_Simulations.html#Setup-a-Keras-Model-for-BER-simulations) of the notebook

```
no = ebnodb2no(ebno_db, self._num_bits_per_symbol, self._coderate, self._rg)
```

which computes the noise power to achieve a desired SNR (or Eb/No to be precise).

**Reply by communicationDL1** [reply]
In this code snippet, no = ebnodb2no(ebno_db, self._num_bits_per_symbol, self._coderate, self._rg) is used to calculate N0. How should one control or maintain a constant N0 or transmit power in this scenario?

**Reply by jhoydis** [reply]
The transmit power is always assumed to be equal to one here. This is because the constellations have unit power. No is chosen according to the desired Eb/No.

**Reply by communicationDL1** [reply]
Thank you for your reply!

**Reply by wjthkx** [reply]
Hello, I am a beginner. Why does BER increase after changing the number of transmit antennas to 16 in this example？
